com.unity.multiplayer.mlapi/Runtime/Core/SnapshotSystem.cs

```diff
@@ -20,16 +20,16 @@ internal struct VariableKey
         public ulong NetworkObjectId; // the NetworkObjectId of the owning GameObject
         public ushort BehaviourIndex; // the index of the behaviour in this GameObject
         public ushort VariableIndex; // the index of the variable in this NetworkBehaviour
+        public ushort TickWritten; // the network tick at which this variable was set
     }
 
     // Index for a NetworkVariable in our table of variables
     // Store when a variable was written and where the variable is serialized
     internal struct Entry
     {
         public VariableKey Key;
-        public ushort TickWritten; // the network tick at which this variable was set
         public ushort Position; // the offset in our Buffer
-        public ushort Length; // the length of the data in Buffer
+        public ushort Length; // the Length of the data in Buffer
         public bool Fresh; // indicates entries that were just received
```

Comment:
Nit, I might re-name this 'Unprocessed' (assuming that's what it means)

Reply:
Yes, but same comment as much of above. This got removed already from the `spawn` branch.